test/base_test.rb

```diff
@@ -15,12 +15,13 @@ def run_command(cmd, refute: false)
 end
 
 def get_component_with_default_values(component)
-  {
-    btx_component_name: "component_name",
-    btx_component_path: "./test/#{component[:btx_component_type]}.metababel_test",
-    btx_component_plugin_name: "plugin_name",
-    btx_compile: true
-  }.update(component)
+  # We have only one component per plugins, the component name can be arbitrary.
+  # TODO: We need to downcase for SOURCE -> source somehwere. They will need to be fixed later
+  { btx_component_name: 'component',
+    btx_component_plugin_name: [component[:btx_component_type],
+                                component[:btx_component_label]].compact.map(&:downcase).join('_'),
+    btx_component_path: "./test/#{component[:btx_component_type]}.#{component[:btx_plugin_name]}.#{component[:btx_component_name]}",
```

Comment:
If we remove c_plugin_name and c_name this will render components folders like `FILTER..`, `SOURCE..` or `SINK..` did you expected that ?

Reply:
Good catch! Fixed